dev_settings.py

```diff
@@ -86,3 +86,7 @@
 SHARED_DRIVE_ROOT = shared_dirname
 
 PHONE_TIMEZONES_SHOULD_BE_PROCESSED = True
+
+ELASTICSEARCH_HOST_PROD = 'hqes0.internal-va.commcarehq.org'
+ELASTICSEARCH_HOST_STAGING = 'hqes0-staging.internal-va.commcarehq.org'
+ELASTICSEARCH_HOST_INDIA = '10.162.36.221'
```

Comment:
was thinking about this a bit last night and thought of a slightly different way to do it that doesn't break the grep test (grepping for `ELASTICSEARCH_HOST_PROD` doesn't find anything in the codebase). The settings could be structured like this:
```py
ELASTICSEARCH_DEBUG_QUERY_HOSTS = {
    'prod': 'hqes0.internal-va.commcarehq.org',
    'staging': 'hqes0-staging.internal-va.commcarehq.org',
    'india': '10.162.36.221',
}
```
It's pretty obvious how the implementation on the other end would need to change. No need to do this if you don't feel it's worth it.